src/Makevars.in

```diff
@@ -77,26 +86,19 @@ ifeq ($(USE_TBB), Windows)
 	# rtools: turn on hacks to compensate for make and shell differences rtools<=>MinGW
 	# compiler: overwrite default (which is cl = MS compiler)
 	MAKE_ARGS += rtools=true compiler=gcc
-	ifeq ($(WIN), 64)
-		MAKE_ARGS += arch=intel64 runtime=mingw
-		ARCH_DIR=x64/
-	else
-		MAKE_ARGS += arch=ia32 runtime=mingw
-		ARCH_DIR=i386/
+	ifneq ($(WINARM64), true)
```

Comment:
Can we add a comment here indicating what the expected default arch / runtime here is when WINARM64 is set? (Or does TBB just know how to do the right thing here?)